packages/amplify-cli/src/commands/env/add.js

```diff
@@ -1,60 +1,10 @@
-const fs = require('fs-extra');
-const path = require('path');
+const { run } = require('../init');
```

Comment:
Do we ask the question if its an existing env when this command is run

Reply:
Yes. We do.